internal/pkg/skuba/deployments/ssh/kubelet.go

```diff
@@ -18,17 +18,156 @@
 package ssh
 
 import (
-	"github.com/SUSE/skuba/internal/pkg/skuba/deployments/ssh/assets"
+	"crypto/x509"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	certutil "k8s.io/client-go/util/cert"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/cmd/kubeadm/app/util/pkiutil"
 
-	"github.com/SUSE/skuba/pkg/skuba"
+	"github.com/SUSE/skuba/internal/pkg/skuba/deployments"
+	"github.com/SUSE/skuba/internal/pkg/skuba/deployments/ssh/assets"
+	skubaconstants "github.com/SUSE/skuba/pkg/skuba"
 	"github.com/SUSE/skuba/pkg/skuba/cloud"
 )
 
 func init() {
+	stateMap["kubelet.rootca.create"] = kubeletCreateRootCert
+	stateMap["kubelet.servercert.create"] = kubeletCreateServerCert
 	stateMap["kubelet.configure"] = kubeletConfigure
 	stateMap["kubelet.enable"] = kubeletEnable
 }
 
+const (
+	// KubeletCertAndKeyDir defines
+	KubeletCertAndKeyDir = "/var/lib/kubelet/pki"
+
+	// KubeletCACertAndKeyBaseName defines kubelet's CA certificate and key base name
+	KubeletCACertAndKeyBaseName = "kubelet-ca"
+	// KubeletCACertName defines kubelet's CA certificate name
+	KubeletCACertName = "kubelet-ca.crt"
+	// KubeletCAKeyName defines kubelet's CA key name
+	KubeletCAKeyName = "kubelet-ca.key"
+
+	// KubeletServerCertAndKeyBaseName defines kubelet server certificate and key base name
+	KubeletServerCertAndKeyBaseName = "kubelet"
+	// KubeletServerCertName defines kubelet server certificate name
+	KubeletServerCertName = "kubelet.crt"
+	// KubeletServerKeyName defines kubelet server key name
+	KubeletServerKeyName = "kubelet.key"
+)
+
+func kubeletCreateRootCert(t *Target, data interface{}) error {
```

Comment:
This might go somewhere else in internal, since it's just a local operation that doesn't need to SSH anywhere. As a rule of thumb, if we are not using the target then it should ideally go somewhere else (not under `deployments/ssh`).

Reply:
Thanks for the reminder, I miss checking the package.
I will create another PR to address these issues.